.github/workflows/test.yml

```diff
@@ -18,22 +18,29 @@ jobs:
     - name: 'Test syntax check'
       run: 'rbs parse gems/**/*.rbs'
   test_on_changed_gems:
+    if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
-    container:
-      image: rubylang/ruby:3.1-focal
```

Comment:
I wanted to use GitHub CLI, so I decided not to use this image. Instead, I configured ruby/setup-ruby.